src/patternfly/layouts/Gallery/styles.scss

```diff
@@ -0,0 +1,7 @@
+.pf-l-gallery {
+  display: grid;
+  grid-gap: 1rem; //revisit this value
+  grid-template-columns: repeat(auto-fill, minmax(250px, 1fr));
```

Comment:
250px is randome but ok for now

Reply:
Yeah. I meant to leave a comment by that line also.